jobs/build/custom/Jenkinsfile

```
@@ -232,6 +232,11 @@ node {
                 if (!any_images_to_build) { return }
                 command = doozerOpts
                 command += "${include_exclude} --profile ${repo_type} images:build --push-to-defaults"
+                if (majorVersion == "4") {
+                    config_dir = "./qe_quay_config"
+                    buildlib.registry_quay_qe_login(config_dir)
+                    command += " --registry-config-dir=${config_dir}"
```

we'll need this here too:
```suggestion
                    command += " --registry-config-dir=${config_dir} --filter-by-os='.*' "
```